library/src/main/java/com/eggheadgames/siren/Siren.java

```diff
@@ -231,13 +245,21 @@ private int checkVersionDigit(String[] minVersionNumbers, String[] currentVersio
     private boolean checkVersionCode(JSONObject appJson) throws JSONException{
         if (!appJson.isNull(Constants.JSON_MIN_VERSION_CODE)) {
             int minAppVersionCode = appJson.getInt(Constants.JSON_MIN_VERSION_CODE);
+            // If no config found, assume version check is enabled
+            Boolean versionCheckEnabled = appJson.has(Constants.JSON_ENABLE_VERSION_CHECK) ? appJson.getBoolean(Constants.JSON_ENABLE_VERSION_CHECK) : true;
```

Comment:
As a small improvement, we can move the logic of getting `Constants.JSON_ENABLE_VERSION_CHECK` value to separate method that returns `boolean`.

Because we have duplication of the logic in the code.

Reply:
Created #30